vcpkg.json

```
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "glvis",
+  "version": "4.1.999",
```

We can probably just put `4.2` here, as the release is imminent 🤞 

```suggestion
  "version": "4.2",
```